cmd/config/types.go

```diff
@@ -33,6 +33,10 @@ const (
 	AuthMethodSessionManager = "session-manager"
 )
 
+const (
+	AnnotationForConfigBypass = "config_bypass"
```

Comment:
```suggestion
	AnnotationForConfigBypass = "config/bypass-check"
```

Reply:
This will keep with the current approach/convention used by output to avoid conflicts between different package's annotations